study-builder/studies/brugada/study-events.conf

```diff
@@ -0,0 +1,435 @@
+        ]
+        "linkedActivityCode": "GENETIC_TESTING",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """
+        user.studies["brugada"].forms["GENETIC_TESTING"].isStatus("COMPLETE")
+      """,
+      "delaySeconds": ${delay.weeks.six},
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 2
+    },
+
+    // GENETIC_TESTING "Completion Thank You " email
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENETIC_TESTING",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.completion_thank_you_en}, "language": "en", "isDynamic": true }
+        ],
+        "pdfAttachments": []
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+
+    // Activity Instance Creation
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "LIFESTYLE"
+      },
+      "preconditionExpr": "true",
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+//    {
+//      "trigger": {
+//        "type": "ACTIVITY_STATUS",
+//        "activityCode": "CONSENT",
+//        "statusType": "COMPLETE"
+//      },
+//      "action": {
+//        "type": "ACTIVITY_INSTANCE_CREATION",
+//        "activityCode": "QUALITY_OF_LIFE"
+//      },
+//      "preconditionExpr": "true",
+//      "maxOccurrencesPerUser": 1,
+//      "dispatchToHousekeeping": false,
+//      "order": 5
+//    },
+
+    //Activity Instance Copy
+    # Instance copy
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "MEDICAL_RECORD_RELEASE",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "COPY_ANSWER",
+        "copyFromPreviousInstance": true,
+        "copyConfigPairs": []
+      },
+      "order": 1,
+      "dispatchToHousekeeping": false
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "BACKGROUND_INFORMATION",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "COPY_ANSWER",
+        "copyFromPreviousInstance": true,
+        "copyConfigPairs": []
+      },
+      "order": 1,
+      "dispatchToHousekeeping": false
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENETIC_TESTING",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "COPY_ANSWER",
+        "copyFromPreviousInstance": true,
+        "copyConfigPairs": []
+      },
+      "order": 1,
+      "dispatchToHousekeeping": false
+    },
+//    {
```

Comment:
can we clean up these commented events if its not needed

Reply:
cleaned up